cranecloud/docs/authentication/login.md

```diff
@@ -0,0 +1,11 @@
+## Login
+1\. Browse this URL http://cranecloud.io/login to access the Crane Cloud login portal, similar to the one below
```

Comment:
Kindly change the URL to the latest URL [<http://staging.cranecloud.io/login>](http://staging.cranecloud.io/login). you could make your links clickable by putting them in <> eg. <http://staging.cranecloud.io/login>.

Reply:
This url has been fixed too @MutegekiHenry